### PR TITLE
Demos: Add basic local demos

### PR DESCRIPTION
**Pull request by ChrisAntaki**
These demos will help developers demo their changes locally

### CODE REVIEW

**Comment by ChrisAntaki** [comment]
Actually, this one needs some more work

File: demos/index.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ * Redirects to SwG Basic JS.
+ * The endpoint is compatible with the classic Swgjs local demos.
+ */
+app.get('/examples/sample-pub/redirect-to/swg-basic.js', (req, res) => {
+  res.redirect('https://news.google.com/swg/js/v1/swg-basic.js');
+});
+
+app.listen(port, () => {
+  console /*OK*/
+    .log(`Swgjs Demos are available at http://localhost:${port}`);
```
Comment (kristenwang) [inline demos/index.js:32]: Should this line here differentiate SwG classic & SwG basic?

Comment (ChrisAntaki) [reply demos/index.js]: Good call, updated

File: demos/public/google541407f935830527.html [hunk 2]
```diff
@@ -0,0 +1 @@
+google-site-verification: google541407f935830527.html
```
Comment (kristenwang) [inline demos/public/google541407f935830527.html:1]: I'm wondering how does the token here gets generated?

Comment (ChrisAntaki) [reply demos/public/google541407f935830527.html:1]: The Google Search Console generates HTML files like these. It helps them verify someone owns a domain or URL path https://support.google.com/webmasters/answer/9008080?hl=en

Comment (ChrisAntaki) [reply demos/public/google541407f935830527.html:1]: The HTML file upload method is the easiest verification method for AppEngine sites

**Review by kristenwang — APPROVED** [review]
LGTM.

**Pull request merged**